edgraph/access_ee.go

```diff
@@ -145,19 +145,17 @@ func (s *Server) authenticateLogin(ctx context.Context, request *api.LoginReques
 	}
 
 	// authorize the user using password
-	var err error
-	user, err = authorizeUser(ctx, request.Userid, request.Password)
+	invalidLogin := errors.New("Invalid username or password")
+	user, err := authorizeUser(ctx, request.Userid, request.Password)
 	if err != nil {
-		return nil, errors.Wrapf(err, "while querying user with id %v",
```

Comment:
Actually, I would return the same error here and log it as glog.Error() and then return the invalidLogin error. This was the alpha logs would have information about why login failed but the user wouldn't know about it.